frontend/src/css/PostRecipe.css

```diff
@@ -1,119 +1,17 @@
-.recipe-container-post-recipe {
-  font-family: 'Arial', sans-serif;
-  font-weight: 600;
-  font-size: 16px;
-  padding: 50px;
-  border-radius: 15px;
-  height: fit-content;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  min-height: 1200px;
-}
-
-.background-image-post-recipe {
+.bg-image {
```

Comment:
If you are using the same background image as before, this is not needed, as currently, in develop, the background image is part of the root.